nightly-style-policy.md

```diff
@@ -0,0 +1,26 @@
+Initial PR(s) implementing new syntax filed against rust-lang/rust should
```

Comment:
make a nice opener that restates that we don't[^1] prescribe nightly style policy

[^1]: until we do, at which time we should make rustfmt follow it

Reply:
jane says something like "if nightly formatting is stated, then we try to follow it"?